nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
- [ ] @Munyabelden great work adding `correct_name`, however you forgot to add a message to be displayed when the method is run, check the example below

```suggestion
 ```ruby
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```
```